libraries/AP_AHRS/AP_AHRS.cpp

```diff
@@ -805,7 +805,7 @@ bool AP_AHRS::airspeed_sensor_enabled(void) const
 
 // return an airspeed estimate if available. return true
 // if we have an estimate
-bool AP_AHRS::_airspeed_estimate(float &airspeed_ret) const
+bool AP_AHRS::_airspeed_estimate(float &airspeed_ret)
```

Comment:
```suggestion
bool AP_AHRS::_airspeed_estimate(float &airspeed_ret) const
{
    Aspd_Status unused_status;
    return _airspeed_estimate(ret, unused_status;
}

bool AP_AHRS::_airspeed_estimate(float &airspeed_ret, Aspd_Status &status) const
```

... and remove the state variable.